build/msw/wx_adv.vcxproj

```diff
@@ -487,7 +480,12 @@
       <ExcludedFromBuild Condition="'$(Configuration)|$(Platform)'=='Release|x64'">true</ExcludedFromBuild>
     </CustomBuild>
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="wx_core.vcxproj">
```

Comment:
`wx_adv` is now a dummy project. It should depend on nothing and nothing should depend on it.